docs/manage/fleet/_index.md

```diff
@@ -104,7 +101,11 @@ Do not make a secret key publicly available, as anyone who has the secret key ca
 It is good practice to note _where_ and _when_ you share a location secret key.
 {{% /alert %}}
 
-### Logs
+### Logging
 
 Each robot automatically sends logs to the cloud where you can view them from the [**Logs** tab](robots/#logs).
-If you are collaborating on a robot and controlling it using the [**Control** tab](robots/#control) or [code](#control-with-code), everyone who has access to the location the robot is in, can see the robot's logs.
+If you are collaborating on a robot and controlling it using the [**Control** tab](robots/#control) or code, everyone who has access to the location the robot is in can see the robot's logs.
+
+### Deployment
+
+You and your collaborators can [deploy control logic](/program/sdks/), [modular resources](/program/extend/modular-resources/), sidecar [processes](../configuration/#processes), or [machine learning models](/services/ml/) to your fleet of robots without manually copying files by uploading it to Viam's cloud and deploying it to your fleet.
```

Comment:
```suggestion
You and your collaborators can deploy [control logic](/program/sdks/), [modular resources](/program/extend/modular-resources/), sidecar [processes](../configuration/#processes), or [machine learning models](/services/ml/) to your fleet of robots without manually copying files by uploading it to Viam's cloud and deploying it to your fleet.
```

Just to have nouns linked, verbs not, in a consistent fashion.

Reply:
Ok, good to know! Done